cucumber-js/_setup_cucumber_runner.js

```diff
@@ -1,5 +1,6 @@
 const Nightwatch = require('../lib');
 const {Before, After, setDefaultTimeout} = require('@cucumber/cucumber');
+const NightwatchState = require('./nightwatchState');
```

Comment:
does this need to be external?

Reply:
Moved `NightwatchState` functions to nightwatch formatter